### DISCUSSION TITLE
Being able to link Prompts with Traces when using the LangChain integration

### DISCUSSION THREAD

**Discussion by jonnyforsterMSG** [langfuse/langfuse#3053] [Ideas]
### Describe the feature or potential improvement

Hi,

Linking a trace or generation with the prompt used is documented [here ](https://langfuse.com/docs/prompts/get-started#link-with-langfuse-tracing-optional) but it's marked unavailable when using Langchain. Any idea when you might make it available as it's something we'd be keen to implement? In the meantime have you got a work around I could use, ideally to populate promptId on the observation but at the very least add it to the trace metadata?

**Comment by marcklingen** [comment]
Unfortunately, this is a known limitation right now, learn more about this here + please add your upvote to the thread: https://github.com/orgs/langfuse/discussions/2237

**Comment by marcklingen** [comment]
We released this feature. Thanks again for raising this.
